example/30-cloudprofile-local.yaml

```diff
@@ -12,3 +12,5 @@ spec:
     constraints:
       dnsProviders:
       - name: unmanaged
+      machineImages:
```

Comment:
You could add this logic into `30-cloudprofile.yaml.tpl`. After executing `./hack/generate-examples` your changes will be removed.
So apply your changes in templates files under `/hack/templates/resources` and run `./hack/generate-examples`.